yarn-project/simulator/src/avm/index.ts

```diff
@@ -1 +1,2 @@
 export * from './avm_simulator.js';
+export * from './journal/index.js';
```

Comment:
I think the journal needs to be renamed StateManager and moved to the src/public/ folder. Will do so in a follow-up PR.